src/Transport/Receiving/RepeatedFailuresOverTimeCircuitBreaker.cs

```diff
@@ -40,7 +41,10 @@ public Task Failure(Exception exception, CancellationToken cancellationToken = d
                 Logger.WarnFormat("The circuit breaker for {0} is now in the armed state", name);
             }
 
-            return Task.Delay(TimeSpan.FromSeconds(1), cancellationToken);
+            //If the circuit breaker has been triggered, wait for 10 seconds before proceeding to prevent flooding the logs and hammering the ServiceBus
+            var delay = triggered ? TimeSpan.FromSeconds(10) : TimeSpan.FromSeconds(1);
```

Comment:
Why do we want to increase the delay only after the circuit has been triggered? Why not always go with `10` seconds delay?

Reply:
Good question. Maybe that would be a good enough solution. What do you think @WojcikMike & @jpalac ?

Reply:
I don't mind that. I was checking other transports, but there are different solutions. So increasing the time to 10 secs is fine

Reply:
Do the other solutions not incorporate a timer? Are they worth looking at to use similar logic or are the transports too different that it's not worth it?

Reply:
Rabbit is using this: https://github.com/Particular/NServiceBus.RabbitMQ/blob/master/src/NServiceBus.Transport.RabbitMQ/Receiving/MessagePumpConnectionFailedCircuitBreaker.cs

And the value is configurable: 
https://docs.particular.net/transports/rabbitmq/connection-settings#controlling-behavior-when-the-broker-connection-is-lost-timetowaitbeforetriggering
Default to 2 mins

The same for SqlServer: 
https://docs.particular.net/transports/sql/connection-settings?version=sqltransport_6#circuit-breaker-wait-time